### PR TITLE
Made required changes. Fixes #687

### LINKED ISSUE #687: Indicator to show if you have the latest version of Plinth
This was mentioned in the last progress call (https://wiki.debian.org/FreedomBox/ProgressCalls/2016-12-18).

After a Plinth release is announced, it's not clear when it will show up in unstable / testing / stable. So users might expect an update to happen, but it's not available for them yet.

In Help -> About, we show the current Plinth version. We can also have an indicator to show if a newer version of Plinth is available, in the configured apt source.

### PR DESCRIPTION
**Pull request by Pk13055**
Made the required changes. Please tell me if something needs to be changed.

### CODE REVIEW

File: plinth/modules/help/help.py [hunk 1]
```diff
@@ -50,9 +50,12 @@ def index(request):
 @public
 def about(request):
     """Serve the about page"""
+    cache = Cache()
+    plinth = cache['Plinth']
```
Comment (jvalleroy) [inline plinth/modules/help/help.py]: The package name is "plinth", lower case.

File: plinth/modules/help/help.py [hunk 2]
```diff
@@ -50,9 +50,12 @@ def index(request):
...
     context = {
         'title': _('About {box_name}').format(box_name=_(cfg.box_name)),
         'version': __version__
+        'new_version': plinth.candidate.is_installed
```
Comment (jvalleroy) [inline plinth/modules/help/help.py]: I get a syntax error here:
```
$ pyflakes3 plinth/modules/help/help.py
plinth/modules/help/help.py:58:21: invalid syntax
        'new_version': plinth.candidate.is_installed
```

File: plinth/modules/help/templates/help_about.html [hunk 3]
```diff
@@ -75,6 +75,9 @@
   <p style='margin-top:30px'>
     {% blocktrans trimmed %}
       You are running Plinth version {{ version }}.
+      {% if {{new_version}} == 'true' %}
```
Comment (jvalleroy) [inline plinth/modules/help/templates/help_about.html]: I think this should just be {% if new_version %}.

**Comment by Pk13055** [comment]
@jvalleroy I've made the requested changes. Sorry for the late response, was out of town.

**Comment by jvalleroy** [comment]
@Pk13055 Thanks, I've merged this change with some additional fixes.

You may want to try setting up vagrant, that is very useful for testing and debugging.

**Pull request closed without merging**